### ISSUE TITLE
cannot import name 'module_config_pb2' from 'meshtastic'

### ISSUE THREAD

**Issue by SpudGunMan** [pdxlocations/meshtastic-curses-client#4]
just updated to .14 python CLI and I suspect related? 

on cloning project and downloading .13 meshtastic python error seen 

`cannot import name 'module_config_pb2' from 'meshtastic'`

**Comment by pdxlocations** [comment]
@ianmcorvidae do you know what the proper import would be now?

**Comment by ianmcorvidae** [comment]
`from meshtastic.protobuf import module_config_pb2`

all the protobufs moved from `meshtastic` to `meshtastic.protobuf` in 2.3.13

**Comment by ianmcorvidae** [comment]
You can also wrap it in a try/catch like this:

```python
try:
    from meshtastic.protobuf import module_config_pb2
except ImportError:
    from meshtastic import module_config_pb2
```

**Comment by SpudGunMan** [comment]
> You can also wrap it in a try/catch like this:
```python
try:
    from meshtastic.protobuf import module_config_pb2
    from meshtastic import BROADCAST_NUM
except ImportError:
    from meshtastic import module_config_pb2, BROADCAST_NUM
```

settings.py also has problems with 
```python
from meshtastic import config_pb2, module_config_pb2, mesh_pb2, channel_pb2
```
not sure where it all is now (didnt look)

**Comment by ianmcorvidae** [comment]
All of the `_pb2` modules are now in `meshtastic.protobuf` as of 2.3.13, so anything that imports those should have the try/catch too, probably

**Comment by SpudGunMan** [comment]
if handy here is a fix https://github.com/pdxlocations/meshtastic-curses-client/pull/5

**Comment by pdxlocations** [comment]
Closed by #5
